src/test/plugin/fixtures/TrustSet.cpp

```diff
@@ -666,24 +967,125 @@ getTransactors()
          61,
          {formatPtr, 5},
          ConsequencesFactoryType::Normal,
-         NULL,
-         NULL,
+         nullptr,
+         nullptr,
+         preflight,
+         preclaim,
+         doApply,
+         nullptr,
+         nullptr,
+         nullptr,
+         nullptr,
+         payFee,
+         consumeSeqProxy,
+         preCompute,
+         apply,
+         checkSingleSign,
+         checkMultiSign},
```

Comment:
You don't need these functions, you can just make them `nullptr`

Reply:
done 3de83f0c1e7e9